backend/dashboard/views/pages/miscellaneous/Error.vue

```diff
@@ -0,0 +1,51 @@
+<template lang="pug">
```

Comment:
Hmm, is this file used anywhere?

Reply:
@taoeffect Yes, all the pages except the landing page is lazy-loaded and in any occasion where the browser has a network connection problem and fails to load the page this 404 page will be presented.

<img src='https://user-images.githubusercontent.com/17641213/231317460-48c33830-85f7-4bf1-a75e-12b4727653d1.png' width='480'>

Reply:
@taoeffect Seeing this again, "404 not found" doesn't sound like a correct description of what has happened. A message like 'Something gone wrong' would be better with 'Retry' button that lets user try to load the page again. :) what's your thought?

Reply:
@SebinSong if the error is for a specific situation, then I would make the description specific too, and maybe rename the file to be more specific as well (since `Error.vue` is very vague, perhaps `ErrorLoading.vue` would be better, allowing you to create other error pages too).

As far as the message, how about in big font, "Error loading page", and in smaller font, "Either our servers are having problems or your connection is down."

Reply:
@taoeffect yeah that sounds good.